Cargo.toml

```diff
@@ -30,21 +30,22 @@ default = [
   "animation",
   "bevy_asset",
   "bevy_audio",
-  "bevy_gilrs",
-  "bevy_scene",
-  "bevy_winit",
   "bevy_core_pipeline",
-  "bevy_pbr",
+  "bevy_gilrs",
   "bevy_gltf",
+  "bevy_pbr",
   "bevy_render",
+  "bevy_scene",
   "bevy_sprite",
   "bevy_text",
   "bevy_ui",
-  "png",
+  "bevy_winit",
+  "filesystem_watcher",
   "hdr",
+  "png",
+  "release_max_level_debug",
```

Comment:
this line is the real change, the rest was just alpha-sorting the list.

Reply:
Why is that not just called `trace` or `tracing` and left out of default features? Negative features are strongly discouraged by cargo itself; all features should be additive.